angular.json

```diff
@@ -3537,6 +3537,126 @@
           }
         }
       }
+    },
+    "design-tokens-ui": {
+      "projectType": "application",
+      "schematics": {
+        "@nrwl/angular:component": {
+          "style": "scss"
+        }
+      },
+      "root": "apps/design-tokens-ui",
+      "sourceRoot": "apps/design-tokens-ui/src",
+      "prefix": "ba",
+      "architect": {
+        "build": {
```

Comment:
rename to `build-frontentd` and add a build tasks like this:

```json

"build": {
  "builder": "@nrwl/workspace:run-commands",
  "options": {
    "parallel":false,
    "commands": [
      {
        "command": "npm run ng run shared-design-tokens:build"
      },
      {
        "command": "npm run ng run design-tokens-ui:build-frontend"
      }
    ]
  }
}
```

Reply:
Furthemore, add an implicit dependency for the design-tokens to the `nx.json`

Reply:
> rename to `build-frontentd` and add a build tasks like this:

This seems to break `ng serve`:
`An unhandled exception occurred: The "path" argument must be of type string. Received undefined`

Reply:
because you have to adapt the `serve` target to use the `build-frontend` and not the `build` target

Reply:
@lukasholzer I see, but in this case we should mention that you should still run `ng build shared-design-tokens` before running the `serve` command, right?

Reply:
you can do the same for the serve rename it to `dev-server` and create a serve target that uses the commands to run the design tokens build first.